scripts/CodeCoverage.py

```diff
@@ -177,6 +207,17 @@ def main(argv):
     USER_WHITELIST = None
     USER_BLACKLIST = None
     GTEST_ZIP_PATH = LAUNCH_DIR + '/3rdparty/gtest-1.7.0.zip'
+    global PROBE_BUILD
+
+    if os.path.exists("/usr/local/probe/bin/cmake"):
+        PROBE_BUILD=True
+        print "Using PROBE_BUILD defaults"
+    else: 
+       PROBE_BUILD=False
+       print "Using /usr/local/ as the default"
+       DEFAULT_BLACKLIST = "/usr/local/.*"
```

Comment:
I think you need to 'import' this global the same way you 'import' `PROBE_BUILD` on line 210 if you want to write to it and have it be preserved.